src/app/event-list/event-list.service.ts

```diff
@@ -70,23 +69,30 @@ export class EventListService {
     private notificationService: NotificationService
   ) {}
 
-  getEvents(activityId?): Observable<any> {
-    let params = {};
-    if (activityId) {
-      params = {
-        type: 'activity_session',
-        activity_id: activityId
-      };
-    } else {
-      params = {
-        type: 'activity_session'
-      };
-    }
-    return this.request.get(api.get.events, {params: params})
-      .pipe(map(response => {
-        return this.normaliseEvents(response.data);
-      })
-    );
+  getEvents(type, activity_Id?) {
```

Comment:
As mentioned above, since we don't have other types now, we should put default type inside this function so that we don't need to pass `activity_session` when we invoke this function